backend/index.ts

```diff
@@ -31,6 +31,10 @@ import homepageController from './controllers/homepage/homepage'
 import deprecatedUrlPage from './controllers/deprecatedUrlPage'
 import requestLimiter from './middleware/requestLimiter'
 
+process.on('unhandledRejection', (reason, promise) => {
+  console.log('Unhandled Rejection at: ', promise, 'reason:', reason)
```

Comment:
Not sure whether we should try to send this to App Insights, if we can - just getting it in there for now